cylc/flow/data_messages.proto

```diff
@@ -106,6 +106,29 @@ message PbWorkflow {
     optional int32 is_runahead_total = 38;
 }
 
+// Selected runtime fields
+message PbRuntime {
+    optional string platform = 1;
+    optional string script = 2;
+    optional string init_script = 3;
+    optional string env_script = 4;
+    optional string err_script = 5;
+    optional string exit_script = 6;
+    optional string pre_script = 7;
+    optional string post_script = 8;
+    optional string work_sub_dir = 9;
+    repeated string execution_polling_intervals = 10;
+    repeated string execution_retry_delays = 11;
+    optional float execution_time_limit = 12;
```

Comment:
I think this is a `PT4S` style value so would that not have to be a string?
https://cylc.github.io/cylc-doc/stable/html/reference/config/workflow.html#flow.cylc[runtime][%3Cnamespace%3E]execution%20time%20limit

Reply:
That's what I thought (and originally had it set to string), however, it appears to be converted into seconds somewhere... (and I believe is used to calculate the progress dial)

Reply:
The string version would be preferable when presenting to the user (for trigger-edit or otherwise), but it's not what we have right at present.

Reply:
I can't find anywhere internally that the string version is used even the `TaskProxy`s define it as float:
```
            execution_time_limit (float):
                Execution time limit of latest job.
```
It's as if it gets converted when the config is read in... maybe a follow-on PR.